sample/src/main/java/com/canhub/cropper/sample/extend_activity/app/SExtendActivityJava.java

```diff
@@ -0,0 +1,35 @@
+package com.canhub.cropper.sample.extend_activity.app;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.canhub.cropper.CropImage;
+import com.canhub.cropper.CropImageActivity;
+
+public class SExtendActivityJava extends CropImageActivity {
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        CropImage.activity()
+                .start(this);
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == CropImage.CROP_IMAGE_ACTIVITY_REQUEST_CODE) {
+            CropImage.ActivityResult result = CropImage.getActivityResult(data);
+            if (resultCode == RESULT_OK) {
+                Uri photoUri = result.getUri();
+            } else if (resultCode == CropImage.CROP_IMAGE_ACTIVITY_RESULT_ERROR_CODE) {
+                Exception error = result.getError();
+                Log.e("err", error.getMessage());
+            }
+        }
+    }
+}
```

Comment:
Why we need this class? what are we testing here that is not tested on `SCameraFragmentJava` ?